src/armsgs.py

```diff
@@ -209,10 +215,17 @@ def warn(self, msg):
             self._log.write(self.cleancolors(premsg+dbgmsg+msg)+"\n")
         return
 
-    def bug(self, msg):
+    def bug(self, msg, close_pdf=None):
         """
         Print a bug message
         """
+        # Close PDF?
```

Comment:
I don't mind this approach, but can't we just grab the logger from within `bug()`

This would save the `import armsgs` in pypit.py, and close_pdf could then become a boolean.